CONTRIBUTING.md

```diff
@@ -96,6 +96,37 @@ package (PyQt5). This can be achieved by running:
   $ pip install pyqt5
 ```
 
+### Using the correct version of spyder-kernels
+
+Following the separation in v3.3 of Spyder's console code into its own package,
+``spyder-kernels``, you'll need to have the corresponding version of it
```

Comment:
This still has double backticks. Please remove them and single backticks.

Reply:
Sorry, my ``push`` failed on an authentication timeout and I didn't notice somehow, so it was still the old version. Should be updated now; let me know if you find anything my ``grep`` and manual search missed.